src/tools/w3dviewer/hierarchyproppage.cpp

```diff
@@ -0,0 +1,84 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief W3D View hierarchy property page
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "hierarchyproppage.h"
+#include "assetmgr.h"
+#include "assetpropertysheet.h"
+#include "meshproppage.h"
+#include "rendobj.h"
+#include "resource.h"
+
+IMPLEMENT_DYNCREATE(CHierarchyPropPage, CPropertyPage)
+
+// clang-format off
+BEGIN_MESSAGE_MAP(CHierarchyPropPage, CPropertyPage)
+    ON_NOTIFY(NM_DBLCLK, IDC_SUBOBJECTLIST, &OnDblClk)
+END_MESSAGE_MAP()
+// clang-format on
+
+CHierarchyPropPage::CHierarchyPropPage(CString &name) : CPropertyPage(IDD_HIERARCHY), m_name(name) {}
```

Comment:
`const CString &name`
because name is not modified

Reply:
fixed.